tasks/testing.rake

```diff
@@ -227,6 +227,20 @@ module Testing
     end
   end
 
+  class HTTPServer
+    def with_server
+      begin
+        server = Process.spawn 'ruby test/opal/http_server.rb'
+        puts 'Waiting for server…'
+        sleep 0.1 until system "curl -s 'http://localhost:4567/' > /dev/null"
```

Comment:
Do we run this suite on AppVeyor? If so this `curl` should be something platform-agnostic.

Reply:
You can check the source code of the chrome runner, it spawns a server in the background. We don't run it on windows, but the code uses something like MRI's `TCPSocket` to check that the server is running.

Reply:
Done! We should probably use it in `SauceLabs.with_server`